### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites_Page Title: Page title is not proper for "Web Sites" page.It is not giving any information regarding "Web Sites" tab This issue is observed through out the application.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#118]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites;  #WCAG2.4.2; #C+E-H; #Chrome; #Edge

**Environment Details:**
**URL:** https://manage.iis.net/web-sites
**Browser Details:** Edge(42.17134.1.0)

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.Tab till "Web Sites" in the left menu section and Press Enter.
6.Verify page title for page.

**Actual Result:**
Page title is not proper for "Web Sites" page. It is not giving any information regarding "Web Sites" tab This issue is observed through out the application.

**Expected Result:**
Page title should be descriptive enough for the user to understand.

**User Impact:** 
Screen Reader users may not know in which page they are in if page title is not provided properly.

**Attachment:**
![bug_118_page title is not descriptive](https://user-images.githubusercontent.com/38523250/44704628-acd34a00-aab9-11e8-96ba-9074a59586d8.png)

**Comment by jhkimnew** [comment]
Won't fix.
I don't see the title tag from other WAC modules. Probably, this is not an issue in WAC environment.

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
